packages/renderer/src/lib/ContainerList.svelte

```diff
@@ -321,7 +321,10 @@ function errorCallback(container: ContainerInfoUI, errorMessage: string): void {
   title="containers"
   subtitle="Hover over a container to view action buttons; click to open up full details.">
   <div slot="additional-actions" class="space-x-2 flex flex-nowrap">
-    <Prune type="containers" engines="{enginesList}" />
+    <!-- Only show if there are containers-->
+    {#if $filtered.length > 0}
```

Comment:
Thinking aloud ...

Wouldn't it be better to use `enginesList.length`? if i start filtering the container and i don't see anything, the button is hidden... but the button is not related to the containers shown.

I mean this behavior could make me think that i'm pruning only the containers that i'm filtering, no?

Reply:
reason being is that we use this as well for the "no containers" view,  so it ensures that whenever "no containers" modal is loaded, that the button is not there either

Reply:
so maybe use of `$containersInfos` (the non-filtered value)

if we have running containers => button is there
if we have containers but filter is filtering out all containers, we still display the button